File: gen_scripts/php/m10.function.php
#!/usr/bin/env php
<?php
// create function
function capitalize($string) {
	return strtoupper($string); // return capitlized string
}

// output string before calling function
$string = "ibm";
echo "The current string is: \"$string\".\n";

// call the function
$result = capitalize($string);  // pass string

# output results after calling function
echo "The capitalized string is: \"$result\".\n";
?>

File: gen_scripts/ruby/m10.function.rb
#!/usr/bin/env ruby

# create method (function)
def capitalize (string)
   string.upcase              # return fully uppercase string
end

# output string before calling method
string = "ibm"
puts "The current string is: \"#{string}\"."

# call the method (function)
result = capitalize(string)

# output results
puts "The capitalized string is: \"#{result}\"."
